mobius-rx/src/main/java/com/spotify/mobius/rx/RxMobius.java

```diff
@@ -57,7 +60,27 @@ private RxMobius() {
    */
   public static <M, E, F> Observable.Transformer<E, M> loopFrom(
       final MobiusLoop.Factory<M, E, F> loopFactory, final M startModel) {
-    return new RxMobiusLoop<>(loopFactory, startModel);
+    return new RxMobiusLoop<>(loopFactory, startModel, Collections.emptySet());
+  }
+
+  /**
+   * Create an observable transformer that starts from a given {@link Init} function.
+   *
+   * <p>Every time the resulting observable is subscribed to, a new MobiusLoop will be started from
```

Comment:
This doesn't read super well. Maybe something like

"Every time the resulting observable is subscribed to, a new MobiusLoop will be started with the initial model and effects resulting from invoking the init function with the start model".

But that makes it seem odd to supply an init function at all. It's easier to just supply the starting model and the initial effects. This behaviour is different than the mobius controller, which retains the last model and uses the init function to restart from that. 

It seems like we should just have a `loopFrom(Factory, M, Set<F>)` method instead? @togi ?

Reply:
I was thinking the same it seems easier to understand to just pass the effects instead of the Init function

Reply:
Done, changes the doc

Reply:
I agree re: passing startEffects. It makes sense to have an `init` if you need to be able to create a new pair of model and start-effects when restarting a paused loop. When you always start from scratch like in this case, it's better to pass in startEffects directly.